wicamo96/FieldScout
# Trends

# Description

Added a trends page which allows you to view a line graph visualizing population trends with a few filters. 

## Type of change


# Testing Instructions

### Create database and populate data if necessary
- If you haven't created the FieldScout database, please do so in visual studio community.
- Use this to fill out tables and columns: [SQL data](https://github.com/wicamo96/FieldScout/blob/main/SQL/01_FieldScout_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/wicamo96/FieldScout/blob/main/SQL/02_FieldScout_Seed_Data.sql)

### Testing Greenhouse Management CRUD
- Click the execute with debugger solid green button to launch the backend in visual studio community.
- From the client directory in a git terminal, run `npm run dev`.
- Once the Vite + React window pops up, you should be directed to the log in page.
- Log in with the default email.
- Navigate to the trends page by clicking the trends link in the navbar
- Select a house to view population trends for bays within that house (House 1 should have enough default values to view trends)
- Select a pest from the pests dropdown and a time frame from the time frame dropdown.
- Optionally select a bay and a bay division.
- Click the view trends button to view the data

# Checklist:
